drivers/at86rf215/Makefile.include

```diff
@@ -1,2 +1,4 @@
+PSEUDOMODULES += at86rf215_timestamp
```

Comment:
```suggestion
PSEUDOMODULES += at86rf215_timestamp
PSEUDOMODULES += at86rf215_subghz
PSEUDOMODULES += at86rf215_24ghz
PSEUDOMODULES += at86rf215_blocking_send
```

Reply:
Those are not present on master, but match `PSEUDOMODULES += at86rf21%`. Maybe we could (independent of this PR) enforce not using wildcards in PSEUDOMODULES? I think it can be helpful to a) be able to list all possible (pseudo-)modules and b) detect typos in `USEMODULE` (which do not happen to collide with existing module names).

Reply:
> Maybe we could (independent of this PR) enforce not using wildcards in PSEUDOMODULES? I think it can be helpful to a) be able to list all possible (pseudo-)modules and b) detect typos in `USEMODULE` (which do not happen to collide with existing module names).

Agreed and should be doable by adding something like a "PSEUDOMODULE .?= .*%.*" check to buildsystem_sanity_check.

Reply:
`NO_PSEUDOMODULE` would then also become unnecessary.

Reply:
In fact, these explicit pseudomodule values are now mandatory, otherwise Murdock is failing.

Reply:
well no, there was a mistake in the pseudomodule definitions, but anyway, I kept the explicit values.

Reply:
I fixed the problem with openmote-b: it was not about default module but rather the NO_PSEUDOMODULE use and a missing pseudomodule (batmon). It should be good now.